Sources/XCTest/Public/XCTestCase.swift

```diff
@@ -195,66 +304,37 @@ open class XCTestCase: XCTest {
     /// class.
     open class func tearDown() {}
 
-    private var teardownBlocks: [() -> Void] = []
-    private var teardownBlocksDequeued: Bool = false
-    private let teardownBlocksQueue: DispatchQueue = DispatchQueue(label: "org.swift.XCTest.XCTestCase.teardownBlocks")
+    private class TeardownBlocksState {
```

Comment:
We’ve lost the thread safety on the list of teardown blocks here. We need to either figure out how to successfully make this an actor again, or (probably easier / better for starting) reintroduce the serial dispatch queue being used as a lock around this state.

Reply:
Oh this is likely a change I unintentionally committed. Let me back this out.

Reply:
I'm starting to get the hang of this whole `actor` concept. 😎  I made quick work of this comment this morning.